### ISSUE TITLE
Make it easier to submit pre-sales questions

### ISSUE THREAD

**Issue by mindctrl** [restrictcontentpro/rcp-site-child#61]
Someone asked on Twitter if we have a pre-sales form. That made me realize it's hard to find.

Here's the flow:
---
1)
![image](https://cloud.githubusercontent.com/assets/1254870/22801868/9366e774-eedc-11e6-944c-6b65bcd4abaa.png)

---
2)
![image](https://cloud.githubusercontent.com/assets/1254870/22801871/9daeaea6-eedc-11e6-8ec2-bc58d8f3cb82.png)

---
3)
scroll down
![image](https://cloud.githubusercontent.com/assets/1254870/22801875/a587385a-eedc-11e6-9a41-799c49c0e3be.png)

---

"Submit a ticket" and "send us a support ticket" do not reveal that there are other options, like pre-sales questions. I think it implies that you want support, as that's what 'tickets' usually are... and it's definitely what 'support tickets' are.

A couple thoughts about the experience...

Opening a pre-sales ticket on the "support" page could be easily misunderstood. If you don't have the product yet, you don't need support. You have questions. With this in mind, it might be sensible to make an entirely different page for pre-sales questions. When people are considering buying, I'd like it to be as easy as possible to ask a question.

If we stick with the same page, or offer the current solution in addition to a new page, I suggest we adjust the phrasing and flow a bit. In this case, I'd love to see a simple two-option choice at the end of the search results. Something like "Ask a pre-sales question" and "Submit a ticket".

Thoughts?

**Comment by pippinsplugins** [comment]
How about adding a grid of options like we did on the EDD [support page](https://easydigitaldownloads.com/support)?

**Comment by mindctrl** [comment]
I think that's a great idea.

**Comment by sumobi** [comment]
Yeah, this is great, we should do it for AffWP also. I could never put a finger on it and now it's blatantly obvious.

**Comment by pippinsplugins** [comment]
👍

**Comment by mindctrl** [comment]
Related to this: I'd love to figure out a flow that ensures people are up to date on RCP (and maybe add-ons). As with all three projects, we get support questions about things we've already fixed. Finding a way to tell people to update, whether that's via a simple notice or parsing the system info file somehow, would be a nice way to help people help themselves.
